mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/SourceOption.scala

```diff
@@ -0,0 +1,14 @@
+package at.forsyte.apalache.infra.passes
+
+/** Defines the data sources supported in the [[PassOptions]] */
+object SourceOption {
+
+  /** Supported data sources */
+  sealed trait T
+
+  /** Data to be loaded from a file */
+  case class File(file: java.io.File) extends T
+
+  /** Data supplied as a string */
+  case class String(content: java.lang.String) extends T
```

Comment:
Can we call this case class something different, e.g., `PlainText` or `TextInput`? I would like to avoid confusion between strings and strings.

Reply:
We can! I assumed the confusion would be avoided by the fact that this class is inside the `SourceOption` object. If I need to disambiguate by some sort of prefix on the class name, I guess I should do this for the `File` class too? Is there any sort of standard convention about how to decide those prefixes?

Reply:
Changed these to `FileSource`  and `StringSource`.